pandas/core/util/hashing.py

```diff
@@ -17,7 +17,7 @@
 from pandas._libs.hashing import hash_object_array
 from pandas._typing import (
     ArrayLike,
-    FrameOrSeriesUnion,
+    DataFrame | Series,
```

Comment:
DataFrame | Series doesn't need importing, just remove the FrameOrSeriesUnion import :)

Reply:
Should I remove all other places where DataFrame | Series is being imported as well?

Reply:
yeah, `from pandas._typing import DataFrame | Series` is invalid syntax anyway